assets/package.json

```diff
@@ -27,6 +27,7 @@
     "@sentry/react": "^7.15.0",
     "@tippyjs/react": "^4.2.6",
     "@types/uuid": "^8.3.4",
+    "bootstrap": "^5.2.3",
```

Comment:
That's a good point. For this one component it was easier to just work with Bootstrap directly, but something more React-native could be nice. My one concern would be whether or not react-bootstrap allows including only part of the Bootstrap CSS in your project, which I think is worth considering given that our static content bundle is already kind of big.

Reply:
My impression is we could still only include the essential pieces of Bootstrap CSS:
https://react-bootstrap.github.io/getting-started/introduction/#stylesheets

> Because React-Bootstrap doesn't depend on a very precise version of Bootstrap, we don't ship with any included CSS